cdc/processor/sinkmanager/table_sink_worker.go

```diff
@@ -329,11 +324,7 @@ func (w *sinkWorker) handleTask(ctx context.Context, task *sinkTask) (finalErr e
 			events = append(events, x...)
 			allEventSize += size
 			usedMem += size
-			if pos.IsCommitFence() {
-				committedTxnSize += size
-			} else {
-				pendingTxnSize += size
-			}
+			pendingTxnSize += size
```

Comment:
Why do we need to change it?

Reply:
In `doEmit` the condition about `CommitFence` is handled correctly. So here it's unnecessary to check `CommitFence` again.